### DISCUSSION TITLE
I have not received education benefits in 6 days

### DISCUSSION THREAD

**Discussion by MichalPlucinski1** [community/community#132348] [GitHub Education]
### Hi everyone,

I have not received education benefits within 5 days of my approval status (I got approval on July 10), and as recomanded, I am raising this discussion

**Comment by RuoyuLiu-f** [comment]
Maybe it's a system delay. Just wait a few days^_^

**Reply by MichalPlucinski1** [reply]
Yeah, Im waiting, my first attempty was like 3 months ago... 
Rising becouse github is telling to do so:

"If you have not received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you. 🌟"

**Comment by eabdalmufid** [comment]
Response #2
